src/components/mainpage/sponsors/single-sponsor.js

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import Img from "gatsby-image"
+import sponsorStyles from "../../../styles/single-sponsor.module.css"
+
+const SingleSponsor = ({ sponsor, type }) => {
+  let assignType
```

Comment:
Couldn't the className be passed directly as a prop?

Reply:
Addressed in the last commit.